config.yaml

```diff
@@ -1,12 +1,10 @@
 name: Spark OrderBook indexer
 networks:
   - id: 0
-    hypersync_config:
-      endpoint_url: https://fuel-17.hypersync.xyz
-    start_block: 8000000
+    start_block: 0
```

Comment:
I didn't put a correct start block, but it takes literally 1 ms more or something to scan from block zero.